src/booking/dto/bookingRes.dto.ts

```diff
@@ -1,15 +1,9 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class BookingResDto {
-  @ApiProperty()
   id: string;
-
-  @ApiProperty()
```

Comment:
Why you removed this decorator, could you explain?

Reply:
Yea, in swagger docs, there written that it will automatically assigned, if you don't need to add additional information you do not need to use it. Now It works as before.